src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -193,12 +201,18 @@ function makeChoroplethStyler(data, colorramp, joinField) {
       },
       getMismatches: function () {
         return [];
+      },
+      getLeafletBounds: function () {
+        return null;
       }
     };
   }
 
   const { min, max } = getMinMax(data);
   const outstandingFeatures = data.slice();
+
+
+  const joinedShapes = [];
```

Comment:
I think this would be a little easier for me to understand if we tracked the bounds here, extended it on each call to `getLeafletStyleFunction`, and just returned the current version in `getLeafletBounds()`.

```js
let bounds = new L.LatLngBounds()
...
bounds = bounds.extend(L.geoJson(shape).getBounds())
...
return bounds;
```